docs/templates/eia860_child.rst.jinja

```diff
@@ -0,0 +1,75 @@
+{% extends "data_source_parent.rst.jinja" %}
+{% block extra_table_text %}
+We’ve also created the following entity tables modeled after EIA data collected from multiple tables.
+{% endblock %}
+
+{% block background %}
+The Form EIA-860 collects utility, owner, plant, and generator-level data from existing
+and planned entities with one or more megawatt of capacity. The form also contains
+information regarding environmental control equipment and construction cost data from
+2013-2018.
+
+As of 2019, the EIA-860 Form is organized into the following schedules:
+
+    * **Schedule 1:** Identification
+    * **Schedule 2:** Power plant data
+    * **Schedule 3:** Generator information
+    * **Schedule 4:** Ownership of generators
+    * **Schedule 6:** Boiler information
+
+(Schedule 5 contained generator construction cost information)
+{% endblock %}
+
+{% block downloadable_pdfs %}
+{% for filename in instruction_files %}
+    * :download:`{{ filename.split("/")[-1].replace("_", " ")[:-4].title() }} (PDF) <{{ filename }}>`
```

Comment:
If `instruction_files` is a list of `Path` objects then I think you could do:
```py
path.stem.replace("_", " ").title()
```
to generate the text for the link here and in the other iterations.